CODEOWNERS

```diff
@@ -329,6 +329,7 @@ extensions/filters/http/oauth2 @derekargueta @mattklein123
 /*/extensions/load_balancing_policies/maglev @wbpcode @nezdolik
 /*/extensions/load_balancing_policies/subset @wbpcode @zuercher @nezdolik
 /*/extensions/load_balancing_policies/cluster_provided @wbpcode @zuercher
+/*/extensions/load_balancing_policies/client_side_weighted_round_robin @wbpcode @adisuissa @efimki
```

Comment:
@adisuissa - it looks like CODEOWNERS must have a write access to the repository, so I'm going to remove myself.

Reply:
We have a few collaborators (non-Envoy maintainers) that are code-owners of the extensions which are there to ensure that future developments comply with the code-owner's view.

Reply:
Sounds good, I'll keep myself in here. :)